awx/ui_next/src/screens/Host/HostList/HostListItem.jsx

```diff
@@ -1,62 +1,99 @@
 import 'styled-components/macro';
-import React from 'react';
+import React, { useState } from 'react';
 import { string, bool, func } from 'prop-types';
 
 import { t } from '@lingui/macro';
 import { Button } from '@patternfly/react-core';
-import { Tr, Td } from '@patternfly/react-table';
+import { Tr, Td, ExpandableRowContent } from '@patternfly/react-table';
 import { Link } from 'react-router-dom';
 import { PencilAltIcon } from '@patternfly/react-icons';
 import { ActionsTd, ActionItem } from '../../../components/PaginatedTable';
 import { Host } from '../../../types';
 import HostToggle from '../../../components/HostToggle';
+import { DetailList, Detail } from '../../../components/DetailList';
+import Sparkline from '../../../components/Sparkline';
 
 function HostListItem({ host, isSelected, onSelect, detailUrl, rowIndex }) {
   const labelId = `check-action-${host.id}`;
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const {
+    summary_fields: { recent_jobs: recentJobs = [] },
+  } = host;
+
+  const recentPlaybookJobs = recentJobs.map(job => ({ ...job, type: 'job' }));
```

Comment:
Are we replacing the type value or adding it in to each object?

Reply:
@jakemcdermott I was adding the type for every object. However this data is already present on the API endpoint.

<img width="546" alt="image" src="https://user-images.githubusercontent.com/9053044/123697539-08eee780-d82b-11eb-84d0-43f5c542402f.png">